designs/developer/index.md

```diff
@@ -15,13 +15,30 @@ Our design documentation covers the entire developer experience in the OpenShift
 -- coming soon --
 
 ## 4.5 Developer Experience
-The following section documents the 4.5 OpenShift Console Developer experience, not just what's new in 4.5
+The following section document the 4.5 OpenShift Console Developer experience.
 
 ### Main Pages
 - [+Add](https://openshift.github.io/openshift-origin-design/designs/developer/add-45/)
+- [Builds](https://openshift.github.io/openshift-origin-design/designs/developer/builds/)
+- [Helm](https://openshift.github.io/openshift-origin-design/designs/developer/helm45/)
+- [Monitoring](https://openshift.github.io/openshift-origin-design/designs/developer/monitoring-44/)
+- [Pipelines](https://openshift.github.io/openshift-origin-design/designs/developer/operator-pipeline-45/pipelines)
+- [Project Access](https://openshift.github.io/openshift-origin-design/designs/developer/project-access-43/)
+- [Search](https://https://openshift.github.io/openshift-origin-design/designs/developer/search-45/)
+- [Topology](https://openshift.github.io/openshift-origin-design/designs/developer/topology-44/)
+
+### Operator enhanced user flows
```

Comment:
@beaumorley this is where a CNV section should be added

Reply:
@serenamarie125 OK. Shall we call this section "CNV" then?

Reply:
Maybe OpenShift Virtualization or just Virtualization? CNV is changing it's name :)

Reply:
Probably general is best then. I'll go with "Virtualization" unless anyone else weights in with concerns. Thanks!

Reply:
@lizsurette I ended up using "OpenShift Virtualization Operator" to match the other 2 Operator enhanced user flows which are named in that way. Just for consistency in naming. I should have looked at those first. LMK if you think this is a problem.

Reply:
I think that's perfect!

Reply:
+1 - should this PR include a similar "coming soon" markdown file at `operator-virtualization-45/index.md` for that future page?